examples/follow_me_example.py

```diff
@@ -0,0 +1,73 @@
+#! /usr/bin/env python3
+
+#This example shows how to use the follow me plugin
+
+import asyncio
+from mavsdk import System
+from mavsdk.follow_me import (Config, FollowMeError, TargetLocation)
+
+default_height = 8.0 #in Meters
+follow_distance = 2.0 #in Meters, this is the distance that the drone will remain away from Target while following it 
+#Direction relative to the Target 
+#Options are NONE, FRONT, FRONT_LEFT, FRONT_RIGHT, BEHIND
+direction = Config.FollowDirection.BEHIND
+responsiveness =  0.02
+
+#This list contains fake location coordinates (These coordinates are obtained from mission.py example)
+fake_location = [[47.398039859999997,8.5455725400000002],[47.398036222362471,8.5450146439425509],[47.397825620791885,8.5450092830163271]]
+
+async def fly_drone():
+    drone = System()
+    await drone.connect(system_address="udp://:14540")
+    
+    #This waits till a mavlink based drone is connected
+    async for state in drone.core.connection_state():
+        if state.is_connected:
+            print(f"-- Connected to drone with UUID: {state.uuid}")
+            break
+    
+    #Checking if Global Position Estimate is ok
+    async for global_lock in drone.telemetry.health():
+        if global_lock.is_global_position_ok:
+            print("-- Global position state is good enough for flying.")
+            break
+    
+    #Arming the drone
+    print ("-- Arming")
+    await drone.action.arm()
+    await asyncio.sleep(1)
+    
+    #Follow me Mode requires some configuration to be done before starting the mode
+    conf = Config(default_height, follow_distance, direction, responsiveness)
+    await drone.follow_me.set_config(conf)
+    await asyncio.sleep(1)
+    
+    print ("-- Taking Off")
+    await drone.action.takeoff()
+    await asyncio.sleep(8)
```

Comment:
Why do we need those `sleep` between actions? :thinking:

Reply:
I guess we do not need it, but i added that there to make sure that drone hovers for some duration after taking off before it actually starts to do something else.. If you feel like we can change it. 
Thanks for your time Jonas..

Reply:
Right. But I don't think it makes sense to sleep between `set_config` and `takeoff` (see line 43), for instance. Also the one at line 67 (waiting until it lands) makes no sense to me. You could just drop it :+1:.